### DISCUSSION TITLE
Use prometheus LLM or other os LLMs through API like Ollama

### DISCUSSION THREAD

**Discussion by arthurGrigo** [prometheus-eval/prometheus-eval#29] [Ideas]
It would be great if one could deploy the judge LLM with a tool like Ollama or use GPT-4 via its API to allow a comparison. This would also allow to remove the LLM dependencies from the lib and allow a simpler installation. I struggle to install prometheus-eval on windows using anaconda.

**Comment by juyoungml** [comment]
Hey @arthurGrigo, thanks for sharing your experience on installing and coming up with this good suggestion. 

I also agree that current `prometheus-eval` should be able to support evaluation through various hosted LLM APIs, not only with local LLMs. I'm working on this in #33 and the feature  will be introduced to version `0.1.4` with the help of `litellm`! This will be done before the end of this week so please stay tuned 🙂
